feature/networkinstance/ate_tests/defaults_test/defaults_test.go

```diff
@@ -131,12 +137,22 @@ func TestDefaultAddressFamilies(t *testing.T) {
 		},
 	}
 	dut := ondatra.DUT(t, "dut")
+	dutP1 := dut.Port(t, "port1")
+	dutP2 := dut.Port(t, "port2")
 	for _, tc := range cases {
 		t.Run(tc.desc, func(t *testing.T) {
+			if *deviations.ExplicitPortSpeed {
+				fptest.SetPortSpeed(t, dutP1)
+				fptest.SetPortSpeed(t, dutP2)
+			}
 			d := &oc.Root{}
 			// Assign two ports into the network instance.
-			assignPort(t, d, dut.Port(t, "port1").Name(), tc.niName, dutPort1)
-			assignPort(t, d, dut.Port(t, "port2").Name(), tc.niName, dutPort2)
+			assignPort(t, d, dutP1.Name(), tc.niName, dutPort1)
+			assignPort(t, d, dutP2.Name(), tc.niName, dutPort2)
+			if *deviations.ExplicitInterfaceInDefaultVRF {
```

Comment:
Let's be more specific `if *deviations.ExplicitInterfaceInDefaultVRF && tc.niName == *deviation.DefaultNetworkInstance`, so that at least this PR changes would be immune to possible test cases ordering changes? What do you think?

Reply:
@xw-g in fact, the current proposal does not depend on the ordering: we need to unassign a port before we can assign to a new VRF in all cases (default->non-default, or non-default->default).

`if *deviations.ExplicitInterfaceInDefaultVRF && tc.niName == *deviation.DefaultNetworkInstance` will work, but it will achieve the inverse target: only default-> non-default test order will work.

Reply:
We would need to `unassignPort` from current vrf, whenever we more ports across vrf, so we need this `unassignPort` call even when `tc.niName == "xyz"`.  Considering that, in future if testcase order changes, current code should work fine.